docker/blackduck-alert/Dockerfile

```diff
@@ -17,6 +17,7 @@ ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
 ENV PATH $JAVA_HOME/bin:$ALERT_TAR_HOME/bin:$PATH
 ENV CERTIFICATE_MANAGER_DIR $ALERT_HOME/bin
 ENV SECURITY_DIR $ALERT_HOME/security
+ENV BLACKDUCK_ALERT_OPTS -Djavax.net.ssl.keyStore=$SECURITY_DIR/blackduck-alert.keystore -Djavax.net.ssl.keyStoreType=PKCS12 -Djavax.net.ssl.keyStorePassword=changeit
```

Comment:
You don't want to hard code the password here.  You will need to evaluate an environment variable because the password can be set via environment variable.
Something like this:
`-Djavax.net.ssl.keyStorePassword=changeit${ALERT_KEY_STORE_PASSWORD:-changeit}`